packages/web3-eth-accounts/src/index.js

```diff
@@ -172,7 +173,7 @@ Accounts.prototype.signTransaction = function signTransaction(tx, privateKey, ca
                         networkId: transaction.networkId,
                         chainId: transaction.chainId
                     },
-                    'petersburg'
+                    transaction.hardfork || "berlin"
```

Comment:
Maybe it's worth mentioning in the release notes that we're now defaulting to `berlin` instead of `petersburg`?

Reply:
I'll update the docs too